modules/oe_theme_helper/oe_theme_helper.services.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
+  oe_theme_helper.twig_loader.namespaces: ['ec-europa', 'ecl']
+  oe_theme_helper.twig_loader.theme: 'oe_theme'
+  oe_theme_helper.twig_loader.directory: 'templates/components'
+services:
+  oe_theme_helper.twig_loader.component_library:
+    class: Drupal\oe_theme_helper\Loader\ComponentLibraryLoader
+    arguments: ['%oe_theme_helper.twig_loader.namespaces%', '@app.root', '%oe_theme_helper.twig_loader.theme%', '%oe_theme_helper.twig_loader.directory%', '@theme_handler']
+    tags:
+      - { name: twig.loader, priority: -100 }
```

Comment:
It's missing a new line at the end.

Reply:
It hasn't been detected by oe-code-review yet, I opened an issue about this: https://github.com/ec-europa/oe-code-review/issues/16

Reply:
In the meantime, I fixed it.